src/adagrams.js

```diff
@@ -1,7 +1,104 @@
+const _ = require('lodash');
+
 const Adagrams = {
+  letters: ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A',
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D',
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E',
+    'F', 'F',
+    'G', 'G', 'G',
+    'H', 'H',
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I',
+    'J',
+    'K',
+    'L', 'L', 'L', 'L',
+    'M', 'M',
+    'N', 'N', 'N', 'N', 'N', 'N',
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q',
+    'R', 'R', 'R', 'R', 'R', 'R',
+    'S', 'S', 'S', 'S',
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U',
+    'V', 'V',
+    'W', 'W',
+    'X',
+    'Y', 'Y',
+    'Z'],
+  scores: {
+    1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+    2: ['D', 'G'],
+    3: ['B', 'C', 'M', 'P'],
+    4: ['F', 'H', 'V', 'W', 'Y'],
+    5: ['K'],
+    8: ['J', 'X'],
+    10: ['Q', 'Z']
+  },
+
   drawLetters() {
-    // Implement this method for wave 1
+    return _.sampleSize(this.letters, 10);
+  },
+
+  usesAvailableLetters(input, lettersInHand) {
+    const playedLetters = (input.toUpperCase()).split('');
```

Comment:
Nice! The parens definitely help with readability, but in case you wanted this to be a slimmer letter, you can rely on the fact that the code gets executed left-to-right, so this is the same code: `const playedLetters = input.toUpperCase().split('');`